hbase-server/src/main/java/org/apache/hadoop/hbase/rsgroup/RSGroupInfoManagerImpl.java

```diff
@@ -825,6 +829,20 @@ private void createRSGroupTable() throws IOException {
     }
 
     public boolean isOnline() {
```

Comment:
I guess the intention here is we will not come back to offline mode after online?

Reply:
@Apache9 any thoughts?

Reply:
The logic seems good to me. @Apache9 Would you like to verify?